lib/ex_aws/sqs/parsers.ex

```diff
@@ -175,6 +179,55 @@ if Code.ensure_loaded?(SweetXml) do
       {:ok, Map.put(resp, :body, parsed_body)}
     end
 
+    def message_attributes_to_map(list_of_message_attribtues) do
+      list_of_message_attribtues
+      |> Enum.reduce(%{}, fn(%{name: name, data_type: data_type} = attr, acc) ->
+        case parse_attribute_value(data_type, attr) do
+          ^attr ->
+            Map.put(acc, name, attr)
+          parsed ->
+            Map.put(acc, name, %{value: parsed, data_type: data_type})
+        end
+      end)
+    end
+
+    defp fix_attributes(parsed_body, attribute_path, fixup_fn) do
+      case get_in(parsed_body, attribute_path) do
+        nil ->
+          parsed_body
+
+        [] ->
+          parsed_body
+
+        v when is_list(v) ->
+          update_in(parsed_body, attribute_path, fixup_fn)
+      end
+    end
+
+    defp parse_attribute_value(<<"String", _rest :: binary>>, %{string_value: string_value}) do
+      string_value
+    end
+
+    defp parse_attribute_value(<<"Binary", _rest :: binary>>, %{binary_value: b64_encoded}) do
+      :base64.decode(b64_encoded)
```

Comment:
I'll change it, but the SQS docs say that all binary data is base64 encoded for you.